types/ramda/index.d.ts

```diff
@@ -1015,33 +1661,97 @@ export function eqBy<T>(fn: (a: T) => unknown): {
 };
 
 /**
- * Reports whether two functions have the same value for the specified property.
+ * Reports whether two objects have the same value, in {@link equals `R.equals`} terms, for the specified property.
+ * Useful as a curried predicate.
+ *
+ * @example
+ * ```typescript
+ * const o1 = { a: 1, b: 2, c: 3, d: 4 };
+ * const o2 = { a: 10, b: 20, c: 3, d: 40 };
+ * R.eqProps('a', o1, o2); //=> false
+ * R.eqProps('c', o1, o2); //=> true
+ * ```
  */
 export function eqProps<T, U>(prop: string, obj1: T, obj2: U): boolean;
 export function eqProps<P extends string>(prop: P): <T, U>(obj1: Record<P, T>, obj2: Record<P, U>) => boolean;
 export function eqProps<T>(prop: string, obj1: T): <U>(obj2: U) => boolean;
 
 /**
- * Returns true if its arguments are equivalent, false otherwise. Dispatches to an equals method if present.
+ * Returns `true` if its arguments are equivalent, `false` otherwise.
  * Handles cyclical data structures.
+ *
+ * Dispatches symmetrically to the `equals` methods of both arguments, if present.
+ *
+ * @example
+ * ```typescript
+ * R.equals(1, 1); //=> true
+ * R.equals([1, 2, 3], [1, 2, 3]); //=> true
+ *
+ * type Recursive = { v: Recursive; };
+ *
+ * const a: Recursive = {} as never; a.v = a;
```

Comment:
What's the intension to assert `{}` to `never`

Reply:
sorry, habit. i'm assuming `as Recursive` will be a lot better?
(i think there's no way to directly construct a `Recursive`)

Reply:
the reason for the habit: it's a lot easier than typing `as <complicated type>` when the type is complicated, and a tiny bit (but not much) safer than `as any`